src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -277,7 +277,7 @@ function BaseSelectionList<TItem extends ListItem>(
     }, [onChangeText]);
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    const debouncedOnSelectRow = useCallback(lodashDebounce(onSelectRow, 1000, {leading: true}), [onSelectRow]);
+    const debouncedOnSelectRow = useCallback(lodashDebounce(onSelectRow, 200), [onSelectRow]);
```

Comment:
This caused https://github.com/Expensify/App/issues/44420. `onSelectRow` was not wrapped in a `useCallback` and as such, it changed whenever the component re-rendered.